design/architecture/architecture.dio.svg

```diff
@@ -1 +1 @@
-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">Object is Literal</div></div></div></foreignObject><text x="105" y="1004" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">Object is Literal</text></switch></g></g><g id="cell-75"><rect x="5" y="680" width="360" height="20" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 358px; height: 1px; padding-top: 690px; margin-left: 6px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font style="font-size: 20px">Multi-Select and Collapsing Ideas</font></div></div></div></foreignObject><text x="185" y="694" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">Multi-Select and Collapsing Ideas</text></switch></g></g><g id="cell-76"><path d="M 30 1050 L 30 1050 L 30 1050 L 50 1060 L 30 1070 L 30 1070 L 30 1070 L 50 1060 Z" fill="#ffffff" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/></g><g id="cell-77"><rect x="70" y="1050" width="370" height="20" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 368px; height: 1px; padding-top: 1060px; margin-left: 71px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">NOTE: in the future, we will want to implement lots of different types of filters for predicates, not just when object is a literal.</div></div></div></foreignObject><text x="255" y="1064" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">NOTE: in the future, we will want to implement lots of differ...</text></switch></g></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://desk.draw.io/support/solutions/articles/16000042487" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
Lets remove this change. We will add updated architecture elements when we have a more complete RDF->LPG workflow.